staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -368,6 +368,9 @@ message PodSandboxConfig {
     map<string, string> annotations = 7;
     // Optional configurations specific to Linux hosts.
     LinuxPodSandboxConfig linux = 8;
+    // Grace period in seconds to wait for the containers in a pod to stop before
+    // forcibly terminating them.
+    int64 termination_grace_period = 9;
```

Comment:
IMHO there's no sense to use negative values here, so maybe `s/int64/uint64/`?

Reply:
Yeah, agree, but my main reason was to match https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L2886.

Reply:
Got it.

From the top of your head, do you know if there's any special meaning for negative values? If not, I think we need to check and error out either here or in cri-o.

Reply:
I have amended the commit to following the minimum grace period that is set for the container stop timeout.